### PR TITLE
COPS-3321 FIRST DRAFT new include file on building a local universe.

### PR DESCRIPTION
**Pull request by SarahStegall**
## Description
https://jira.mesosphere.com/browse/COPS-3321

MOVED/RENAMED FROM ss/DCOS-3321

### CODE REVIEW

File: pages/services/edge-lb/data.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+packageName: edge-lb
+serviceName: edgelb
+techName: Edge-LB
+techShortName: Edge-LB
+
+# Values specific to certain templates:
+
+install:
+  minNodeCount: three
+  nodeDescription: with three Apache Cassandra nodes
```
Comment (vespian) [inline pages/services/edge-lb/data.yml]: Why do we mention Cassandra here?

File: pages/services/include/build-local-universe-ee-only.tmpl [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+
+1. If you are using the latest stable release of DC/OS, download the [local-universe](https://downloads.mesosphere.com/universe/public/local-universe.tar.gz) container to each of your masters. Otherwise, you will need to build the container for your DC/OS version. See the section on [Building Your Own](#building-your-own) and copy it to each of your masters.
+
+1. Load the container into the local docker instance on each master:
+
+    ```bash
+    $ docker load < local-universe.tar.gz
+    ```
+
+1. Add the [`dcos-local-universe-http.service`](dcos-local-universe-http.service) definition to each of your masters at `/etc/systemd/system/dcos-local-universe-http.service` and then start it.
```
Comment (vespian) [inline pages/services/include/build-local-universe-ee-only.tmpl]: where can I find this file?

File: pages/services/include/build-local-universe-ee-only.tmpl [hunk 3]
```diff
@@ -0,0 +1,311 @@
...
+1. Place your CLI and image resources on a web server accessible to CLI and web UI users.
+
+2. Edit the URLs in the `resource.json` file of your package of interest to point to each of those resources, in the `images` and `cli` sections.
+
+3. Edit Makefile so the call to `local-universe.py` includes arguments `--nonlocal_images` and `--nonlocal_cli`.
+
+4. Proceed with [Building Your Own](#building-your-own) as above.
+
+<a name="running-local"></a>
+## Running Local Universe as a Marathon Service
```
Comment (vespian) [inline pages/services/include/build-local-universe-ee-only.tmpl]: Now I am at loss - are we describing here the way to add Edgelb and other enterprise-only packages to a local universe or are we describing how to build local universe? Maybe we should split the part about building local universe into a separate document and concentrate here only on the issue described in COPS-3321?

This comment also applies to two previous sections about building the universe package.

File: pages/services/include/build-local-universe-ee-only.tmpl [hunk 4]
```diff
@@ -0,0 +1,311 @@
...
+      "UNIQUE"
+    ]
+  ]
+}
+```
+
+<a name="making-changes"></a>
+## Making changes...
+
+###  ...to the `universe-static` image
```
Comment (vespian) [inline pages/services/include/build-local-universe-ee-only.tmpl]: This is just a copy-paste of the GitHub documentation - what do we want to achieve here by splitting up this paragraph heading into two and using an ellipsis?

File: pages/services/include/build-local-universe-ee-only.tmpl [hunk 5]
```diff
@@ -0,0 +1,311 @@
...
+1. Bump the default value of `static_version` in `Makefile`
+1. Create a pull request with the above changes
+1. Once it is merged, run the [Publish universe-static image](https://teamcity.mesosphere.io/viewType.html?buildTypeId=Oss_Universe_2_PublishUniverseStaticImage&branch_Oss_Universe_2=%3Cdefault%3E&tab=buildTypeStatusDiv) build, providing the new version number when prompted
+1. Submit a pull request that updates the `FROM universe-static:...` line in
+`Dockerfile.static.base` to use the new version
+
+<a name="adding-stubs"></a>
+## Adding Stub Universes (Custom Universe Packages)
+
+If you've been provided stub universe json files (such as for {{ model.techName }}), you can add them to the local universe with the `add-stub-universe.sh` script.
```
Comment (vespian) [inline pages/services/include/build-local-universe-ee-only.tmpl]: We do not specify where we can get the `add-stub-universe.sh` file.

**Comment by SarahStegall** [comment]
Pawel;
If you take a look at the original dialogue for COPS-3321, you'll see that this is an adaptation of a document Justin Lee wrote on Github for Docker (https://github.com/mesosphere/universe/tree/version-3.x/docker/local-universe#adding-stub-universes-custom-universe-packages). Craig added a few comments in a private Slack channel, and there's some more discussion. I tried to add you to the Slack channel so you could see it, but it kept not adding you. So I have copied the entire conversation into a text file, which I am appending. Hopefully this will answer your questions.
[Slack channel conversation COPS-3321.txt](https://github.com/mesosphere/dcos-docs-site/files/2146978/Slack.channel.conversation.COPS-3321.txt)

File: pages/services/edge-lb/data.yml (hunk 1 above)
Comment (SarahStegall) [reply pages/services/edge-lb/data.yml]: This file was copied from Cassandra. I have not modified it, since I am not the expert here.

**Pull request closed without merging**